docs/content/validate_types.fsx

```diff
@@ -5,29 +5,34 @@
 #r "C:/Program Files (x86)/Reference Assemblies/Microsoft/Framework/.NETFramework/v4.6.1/Facades/System.Runtime.dll"
 
 (**
-Validating Views and Mutables
+Validating Signals and Mutables
 ========================
 
-Views and Mutables in Gjallarhorn also provide a mechanism to filter based on validation rules.
-
+Signals and Mutables in Gjallarhorn also provide a mechanism to filter based on validation rules.
 A review of the [basics of validation in Gjallarhorn](validation.html) prior to reading this section will be beneficial.
+The validation engine in Gjallarhorn can be used to validate Signals and Mutables directly.
+
+`Signal.validate` allows you to filter a Signal into an `IValidatedSignal<'a,'a>`, which provides the following new properties:  
```

Comment:
Signal.validate will create IValidatedSignal<'a,'b> - it's possible to change the type in the validation.  The xamarin samples do this.